.github/workflows/release.yml

```diff
@@ -63,29 +62,3 @@ jobs:
           NODE_AUTH_TOKEN: ${{ secrets.NODE_AUTH_TOKEN }}
           NPM_CONFIG_PROVENANCE: true
         run: pnpm publish --filter "./v-next/**" -r --no-git-checks --tag next --access public
-
-      - name: Check if release was published
-        id: after
-        if: steps.before.outputs.released == 'false'
-        run: node scripts/check-release.mjs
-
-      - name: Prepare GitHub release
-        id: release
-        if: steps.before.outputs.released == 'false' && steps.after.outputs.released == 'true'
-        run: node scripts/prepare-github-release.mjs
-
-      - name: Create GitHub Release
-        if: steps.before.outputs.released == 'false' && steps.after.outputs.released == 'true'
-        env:
-          GITHUB_TOKEN: ${{ github.token }}
-        uses: galargh/action-gh-release@571276229e7c9e6ea18f99bad24122a4c3ec813f # https://github.com/galargh/action-gh-release/pull/1
-        with:
-          # We want the GitHub releases to be verified and published by a human because they are automatically pulled into the website
-          draft: true
-          tag_name: hardhat@${{ steps.release.outputs.version }}
-          generate_release_notes: false
-          target_commitish: ${{ github.sha }}
-          make_latest: ${{ steps.release.outputs.latest == 'true' }}
-          prerelease: ${{ steps.release.outputs.prerelease == 'true' }}
-          body: ${{ steps.release.outputs.body }}
-          token: ${{ github.token }}
```

Comment:
We will revert to manually creating GitHub releases. We expect a release per package.
We can revisit this in the near future.

Reply:
Do we want a GitHub release for each package or just hardhat?

Reply:
We want a release for each package.